ui-components/public/images/DAHLIA-logo.svg

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It seems odd to me to add a DAHLIA-specific logo to ui-components, was this intentional? Just for testing purposes?

Reply:
Oop seeing your note in the PR description now - I would personally prefer to keep the logo as it was or to add a generic one (but that seems like more effort than it is worth)

Reply:
yeah it just seemed like a more realistic example than the listing image 😕 . I can update to put it back to the listing image now that I've tested it out

Reply:
I'll make a ticket on our end to get some better generic testing SVGs in there :)

Reply:
pushed!